### PR TITLE
Change branding preview text

### LINKED ISSUE #9740: Generalise preview in Branding
## Describe the bug
The current contrast example uses the DEV logo.

## To Reproduce
Go to /settings
Scroll down to "Branding"
See the DEV logo.

## Expected behaviour
There should be a general icon/button on display or the forem owner should be able to choose their own SVG.

## Screenshots
branding

Desktop (please complete the following information):

OS: Windows 10 (1909)
Browser: Chrome 84.x

## Additional context
Spotted on https://forem.dev

### PR DESCRIPTION
**Pull request by jeansberg**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Hello! This is my first PR in this repo and it's intended to resolve #9740. It changes the branding preview.

image

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

alt_text

### CODE REVIEW

File: app/views/shared/_logo_design.html.erb [hunk 1]
```diff
@@ -24,10 +24,26 @@
...
     <p class="text-uppercase color-base-50 mb-4 fs-s ff-accent">Preview:</p>
-    <svg id="color-select-preview-logo" class="p-4 radius-default" xmlns="http://www.w3.org/2000/svg" version="1" width="128" viewBox="0 0 132.000000 65.000000" role="img">
+    <svg id="color-select-preview-logo" class="p-4 radius-default" xmlns="http://www.w3.org/2000/svg" version="1" width="128" viewBox="0 0 124.000000 36.000000" role="img">
       <title id="aldzs1ucwkayuyoa6rxtxo2ikd8bpr0q">App logo</title>
-      <path d="M0 33v32h11.3c12.5 0 17.7-1.6 21.5-6.5 3.8-4.8 4.4-9 4-28-.3-16.8-.5-18.2-2.7-21.8C30.3 2.5 26.1 1 12 1H0v32zm23.1-19.1c2.3 1.9 2.4 2.3 2.4 18.5 0 15.7-.1 16.7-2.2 18.8-1.7 1.6-3.5 2.2-7 2.2l-4.8.1-.3-20.8L11 12h4.9c3.3 0 5.6.6 7.2 1.9zM46.1 3.6c-2 2.6-2.1 3.9-2.1 29.6v26.9l2.5 2.4c2.3 2.4 2.9 2.5 16 2.5H76V54.1l-10.2-.3-10.3-.3v-15l6.3-.3 6.2-.3V27H55V12h21V1H62.1c-13.9 0-14 0-16 2.6zM87 15.2c2.1 7.9 5.5 20.8 7.6 28.8 3.2 12.3 4.3 15 7 17.7 1.9 2 4.2 3.3 5.7 3.3 3.1 0 7.1-3.1 8.5-6.7 1-2.6 15.2-55.6 15.2-56.8 0-.3-2.8-.5-6.2-.3l-6.3.3-5.6 21.5c-3.5 13.6-5.8 20.8-6.2 19.5C105.9 40 96 1.9 96 1.4c0-.2-2.9-.4-6.4-.4h-6.4L87 15.2z">
-      </path>
+      <g
```
Comment (jeansberg) [inline app/views/shared/_logo_design.html.erb:29]: How do I generate a single path instead of all this stuff? 🙏

Comment (rhymes) [reply app/views/shared/_logo_design.html.erb:29]: I think we should put this SVG in a file, which is going to be cleaned by the SVG optimizer which runs on commit.

To include a SVG file you can use `inline_svg_tag`, see 

https://github.com/forem/forem/blob/3401d101f73285186c337dcf2f92f5e175ffb095/app/views/shared/authentication/_providers_sidebar.html.erb#L6

for example

**Comment by rhymes** [comment]
@benhalpern @ludwiczakpawel do you know where that logo is in the repo?

**Comment by jeansberg** [comment]
> I like the work, but how about we switch to an SVG version of the 🌱 symbol (Currently the _temporary_ forem logo)

Sounds good. I found seedling.png in the repo, but no SVG version.

**Comment by ludwiczakpawel** [comment]
@jeansberg 

I think you could use this one:

```svg
<svg width="162" height="88" viewBox="0 0 162 88" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M9.9 55.245v18.088c0 3.89 1.565 7.62 4.35 10.371A14.946 14.946 0 0024.751 88H67v-9.778H24.751a4.982 4.982 0 01-3.5-1.432 4.858 4.858 0 01-1.45-3.457V53.29a9.696 9.696 0 00-1.878-5.732A9.894 9.894 0 0013 44a9.893 9.893 0 004.924-3.557 9.696 9.696 0 001.878-5.732V.84a14.876 14.876 0 00-5.55 3.457 14.575 14.575 0 00-4.35 10.37v18.09c0 .963-.193 1.916-.566 2.806a7.327 7.327 0 01-1.61 2.379 7.431 7.431 0 01-2.408 1.59c-.901.368-1.867.558-2.842.558H1a1 1 0 00-1 1v5.822a1 1 0 001 1h1.475c1.97 0 3.858.773 5.25 2.148a7.288 7.288 0 012.176 5.186zM152.067 55.244v18.09c0 3.89-1.569 7.62-4.364 10.37A15.018 15.018 0 01137.168 88H97.202v-9.778h39.966a5.007 5.007 0 003.512-1.432 4.852 4.852 0 001.455-3.457V53.29a9.667 9.667 0 011.884-5.732A9.925 9.925 0 01148.958 44a9.925 9.925 0 01-4.939-3.557 9.667 9.667 0 01-1.884-5.732V14.667c0-1.297-.524-2.54-1.455-3.457a5.007 5.007 0 00-3.512-1.432H85.5C83.5 5.867 79 1.63 77 0h60.168c3.952 0 7.741 1.545 10.535 4.296a14.548 14.548 0 014.364 10.37v18.09c0 .963.193 1.916.567 2.806.375.89.923 1.698 1.615 2.379a7.46 7.46 0 002.417 1.59 7.555 7.555 0 002.851.558H161a1 1 0 011 1v5.822a1 1 0 01-1 1h-1.483a7.51 7.51 0 00-5.268 2.148 7.276 7.276 0 00-2.182 5.185z" fill="#08090A"/><path d="M49.2 0c14.842 0 27.437 9.58 31.886 22.866-7.2 6.092-11.952 14.945-12.61 24.912H63.6c-18.557 0-33.6-14.974-33.6-33.445V1a1 1 0 011-1h18.2zM129 21.111a1 1 0 011 1v10.945c0 17.152-13.968 31.055-31.2 31.055h-12V88h-9.6V54.556l.091-4.778C78.515 33.744 91.98 21.11 108.4 21.11H129z" fill="#1AB3A6"/></svg>
```

This is the logo we currently have in the header of forem.dev. So maybe just use the leaves?

**Comment by Link2Twenty** [comment]
Would it be better to have a more generic file name so people can change the SVG in their own Forem `app/assets/images/forem-logo.svg` to become something like `app/assets/images/preview-logo.svg `?

**Comment by ludwiczakpawel** [comment]
@Link2Twenty we will soon step away from using logo itself in the preview and we will replace it with kinda _wireframed_ preview like this: 

image

So I wouldn't put too much effort into turning that into it's own file.

**Comment by jeansberg** [comment]
> @Link2Twenty we will soon step away from using logo itself in the preview and we will replace it with kinda _wireframed_ preview like this:
> 
> image
> 
> So I wouldn't put too much effort into turning that into it's own file.

Oops, I already changed the filename 😃

**Comment by jeansberg** [comment]
> I like the work, but how about we switch to an SVG version of the 🌱 symbol (Currently the _temporary_ forem logo)

Can I get a re-review? :)

**Pull request merged**